examples/linear_model/plot_ridge_coeffs.py

```diff
@@ -0,0 +1,82 @@
+"""
+==============================================================
+Plot Ridge coefficients as a function of the regularization II
+==============================================================
+
+.. currentmodule:: sklearn.linear_model
+
+:class:`Ridge` Regression is the estimator used in this example.
+Each color in the left plot represents a different feature of the
+coefficient vector, and this is displayed as a function of the
+regularization parameter.
+
+In this example the dependent variable Y is set as a function
+of the input features: Y = Xa + bias. The coefficient vector a is
+randomly sampled from a normal distribution, as well as the bias
+term.
+
+As alpha tends toward zero the coefficients found by Ridge
+regression stabilize towards the randomly sampled vector a.
+For big alpha (strong regularisation) the coefficients
+are smaller (eventually converging at 0) leading to a
+simpler and biased solution.
+These dependencies can be observed on the left plot.
+
+The right plot shows the mean squared error between the
```

Comment:
I don't think it is necessary. But you can try, you will get a very similar curve I suppose, except that it might be shifted upwards, since for no reglarization you should now get some error on the coefficients.